packages/shared/storybook-config/preview.ts

```diff
@@ -26,10 +26,36 @@ const findHtmlNode = (node: serializedNodeWithId): serializedNodeWithId | undefi
   return undefined;
 };
 
-const renderToCanvas: RenderToCanvas<RRWebFramework> = async (context, element) => {
+function snapshotFileName(snapshotId: string, viewport: string) {
+  const fileNameParts = [snapshotId, viewport, 'snapshot.json'];
+  return fileNameParts.join('.');
+}
```

Comment:
This is duplicated from `snapshot-files.ts`, right? I assume for bundling reasons you wanted to avoid importing here. Might be worth comment explaining that (and maybe in the other file so we don't forget this if we change it).

Reply:
Good call, added comments. I think we can make the bundling work, but will tackle that separately at some point.